app/util/popup.ts

```diff
@@ -0,0 +1,50 @@
+export default {
+  open: (url: string, timeout = 60, width = 400, height = 600) => {
```

Comment:
(I don't have much experience with popup windows)

Does the timeout mean the user only gets 1 minute to log in and then the popup closes? Wondering if we should make it a tad longer like 5 min or something

Reply:
I took the 60s timeout from the Auth0 library, but I agree it feels a bit short. Bumped it up to 5 min.